zerver/views/typing.py

```diff
@@ -60,3 +67,42 @@ def send_notification_backend(
         check_send_typing_notification(user_profile, user_ids, operator)
 
     return json_success(request)
+
+
+@has_request_variables
+def send_message_edit_notification_backend(
+    request: HttpRequest,
+    user_profile: UserProfile,
+    operator: str = REQ("op", str_validator=check_string_in(VALID_OPERATOR_TYPES)),
+    message_id: Optional[int] = REQ(json_validator=check_int, default=None),
+) -> HttpResponse:
+    if message_id is None:
+        raise JsonableError(_("Missing message_id"))
```

Comment:
Just remove the `default=None` parameter that makes this parameter not required, and you can delete this check -- the `REQ` framework is designed to take care of this sort of thing for you.

Reply:
updated.